### PR TITLE
Fix node.js.yml CI

### PR DESCRIPTION
**Pull request by kellymore**

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
 
 name: Node.js CI
 
 on:
   push:
-    branches: [main]
+    branches: [ $default-branch ]
   pull_request:
-    branches: [main]
+    branches: [ $default-branch ]
```
Comment (ShreyaThapa) [inline .github/workflows/node.js.yml:11]: Is there any significance of using `$default-branch` instead of `main`?

Comment (kellymore) [reply .github/workflows/node.js.yml:11]: Just best practices

Comment (ShreyaThapa) [reply .github/workflows/node.js.yml:11]: Got it.

**Review by ShreyaThapa — APPROVED** [review]

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (ShreyaThapa) [reply .github/workflows/node.js.yml:11]: Let's merge this in and test with a small PR.

**Pull request merged**